markdown/org/docs/designs/shelly/instructions/en.md

```diff
@@ -0,0 +1,92 @@
+   - Up to 80% or so will produce a crop top.
+   - 100% ends at the waistband of most pants, with no overlap.
+   - 120% will produce a typical t-shirt.
+   - 140% will produce a somewhat longer shirt that is less prone to untucking or riding up.
+   - 200%+ can be used to make a simple t-shirt dress. It's suggested that you set straight sides to false, and adjust the side shape (under advanced options) for t-shirt dresses.
+ - If in doubt, it's a lot easier to shorten a shirt or a sleeve than to lengthen it.
+
+ - Figure out what kind of fit you want and set your eases accordingly. Make sure you have an appropriate fabric. If in doubt, making the shirt too loose is safer than too tight.
+   - In general, when working with spandex, you want 0% to negative ease. When working with ordinary cotton/polyester t-shirt fabric, you want positive ease, both because the fabric stretches less, and because typical styles using that fabric are looser.
+   - Sensory compression shirts will be the tightest / have the most negative ease.
+     - 4-way stretch fabric is required. Swim fabric or another very stretchy fabric is strongly recommended.
+	 - Chest ease: -30% to -20%. This is the most important part to compress for sensory purposes, and is a relatively safe place to put tension.
+	 - Sleeve ease: -20% to -10%. Compression here can be beneficial, but you don't want it so tight that you cut off circulation.
+	 - Wrist ease: -15% to 0%. Mostly preference.
+     - Neck ease: 25% to 100% (25% will still be a very snug neckband that will take some stretching to get on/off the head, because of how the neckband is constructed and because neckholes usually are quite a bit bigger than the neck to fit over the head.
+
+   - Athletic shirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Highly breathable fabric with some spandex/stretch is recommended.
+	 - Eases are somewhere between those for a compression shirt and a swim shirt.
+
+   - Swim shirt (looser fitting stretch shirt meant for swimming)
+     - Swim fabric (spandex/nylon blend or spandex/polyester blend, around 20% spandex)
+     - Chest ease: ~0%
+	 - Sleeve ease: 0-15%
+	 - Neck ease: 50 to 150% (mostly a style decision). Ease under 50% isn't recommended unless you like very snug neckbands.
+
+   - T-shirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Eases will be quite a bit more larger (more positive) than for other shirt types.
+
+   - Sweatshirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Eases will be similar to that for a t-shirt, or a little larger.
+
+ - The neckband length will be mostly determined by the material of the neckband. Stretch fabric will generally use a length of around 80%. Less stretchy fabric will be longer, but always shorter than 100%, and ribbed knit (traditional t-shirt neckbands) are around 60-70%.
```

Comment:
I would say ribbed knit is more of a recommendation than it being traditional for T-shirts. I believe the alt is same fabric with 1cm - 1.5cm taken off of the neck hole length but that may just be my opinion.

Reply:
1.0 - 1.5 cm sounds awfully little. I'm going to replace 'traditional' with 'recommended,' but I'm not going to mention any absolute amount of stretching.